src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -104,47 +119,124 @@
+  (let [view-type (reagent/atom :card-view)
+        selected-tab (reagent/atom :all)
+        featured-communities-count (count featured-communities)
+        scroll-component (scroll-page/scroll-page
+                          {:background-color  (colors/theme-colors
+                                               colors/white
+                                               colors/neutral-95)
+                           :content-container (fn []
+                                                [rn/view {:margin-top 112}
+                                                 [discover-communities-header
+                                                  {:featured-communities-count featured-communities-count
+                                                   :featured-communities       featured-communities
+                                                   :view-type                  @view-type
+                                                   :selected-tab               selected-tab}]])
+                           :name              (i18n/label :t/discover-communities)})]
+    (fn []
+      (let [sticky-header  (memoize (render-sticky-header selected-tab))
+            page-component (memoize (render-page-content selected-tab @view-type))]
+        (fn []
+          (scroll-component
+           sticky-header
+           page-component))))))
 
 (defn discover
   []
-  (let [view-type (reagent/atom :card-view)]
-    (fn []
-      (let [communities                (rf/sub [:communities/sorted-communities])
-            featured-communities       (rf/sub [:communities/featured-communities])
-            featured-communities-count (count featured-communities)]
-        [safe-area/consumer
-         (fn []
-           [rn/view
-            {:style {:margin-left      20
-                     :margin-right     20
-                     :flex             1
-                     :background-color (colors/theme-colors
-                                        colors/white
-                                        colors/neutral-90)}}
-            [quo/button
-             {:icon     true
-              :type     :grey
-              :size     32
-              :style    {:margin-vertical 12}
-              :on-press #(rf/dispatch [:navigate-back])}
-             :i/close]
-            [screen-title]
-            [featured-communities-header featured-communities-count]
-            [featured-list featured-communities @view-type]
-            [other-communities-list communities @view-type]])]))))
+  (fn []
+    [safe-area/consumer
+     (fn []
+       (let [featured-communities (rf/sub [:communities/featured-communities])]
+         [rn/view
+          {:style {:flex             1
+                   :background-color (colors/theme-colors
+                                      colors/white
+                                      colors/neutral-95)
+                   :position         :absolute
+                   :top              (if platform/ios? 0 44)
```

Comment:
i've this hack, cc @vkjr

Reply:
definitely someone should explain why we are doing this

Reply:
it's just the position of the absolute screen lands differently for ios and android. I can't say of the exact use here but all of these pages that are using the scroll page component will have this... although it would be best to just only have this set in the scroll page so we don't need to set it in all the individual pages.. e.g discover, join, messages etc.


Also @jo-mut  I noticed above you are using safe-area/consumer... is that needed? can it be removed?

Reply:
`safe-area/consumer` doesnt  really have any effect. Instead of using height and width 100%. It works just okay with `top 0 bottom 0 left 0 and right 0`